go/fix-chain/main/fix-chain.go

```diff
@@ -0,0 +1,116 @@
+package main
```

Comment:
When you build it with `go build`, but not with `go get` or `go install`. To experience this for yourself, try this:

```sh
cd $GOPATH
go install github.com/google/certificate-transparency/go/fix-chain/main/
ls bin
```